docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Do you want to **report a bug?** or **request a feature?** 
+
+## Report a bug
+
+Please provide the steps to reproduce and if possible a reproductible environment. **As we are working directly with container, please provide the Dockerfile sample or the Docker image name**
+
+* [ ] **Explain with a simple sentence the behavior expected**
+* [ ] **Tartiflette version:** _e.g 0.1.0_
+* [ ] **Python version:** _e.g 3.6_
+* [ ] **Executed in docker:** _e.g Yes_
+* [ ] **Dockerfile sample:** _Link of sample_
+* [ ] **GraphQL Schema & Query:** _e.g gist, pastebin or directly the query_
+* [ ] **Is a regression from a previous versions?** _e.g Yes_
+* [ ] **Stacktrace** _e.g gist, pastebin..._
+
+## Request a feature
+
+* A feature is missing from the GraphQL Specification?
+* A hook is missing in the API to plug your code?
+* A part of the project could be improved?
+
+Don't hesitate to ask a new feature or code it by yourself.
```

Comment:
Don't hesitate to ask a new feature or join in and send us a pull request with your own feature !